File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] result = new int[2];
        for (int i = 0; i < nums.length - 1; i++) {
            for (int t =i+1; t < nums.length; t++) {
                if ((nums[i] + nums[t]) == target) {
                    result[0] = i;
                    result[1] = t;
                    return result;
                }
            }
        }
        return new int[]{-1, -1};
    }
}

File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        #Dictionray to store seen numbers and their indexes
        num_dic = {}
        #Enumerating through the nums list
        for i,num in enumerate(nums):
            #calculating diff for each num
            diff = target - num
            #if diff is in num dictionary 
            if diff in num_dic:
                #we call the index of the diff and current index
                return [num_dic[diff],i]
            #else we store the current number with index
            num_dic[num] = i
        #In case no solution found
        return []